src/trio/_tests/test_highlevel_open_unix_stream.py

```diff
@@ -11,10 +11,11 @@
 
 assert not TYPE_CHECKING or sys.platform != "win32"
 
-if not hasattr(socket, "AF_UNIX"):
-    pytestmark = pytest.mark.skip("Needs unix socket support")
+has_unix = hasattr(socket, "AF_UNIX")
+reason_unix = "Needs unix socket support"
```

Comment:
Instead of splitting these out, we could just store the `skipif` call here unconditionally, then reuse it lower down.

Reply:
ah of course, thanks!